app/lib/actions/pulp/abstract_async_task_group.rb

```diff
@@ -0,0 +1,81 @@
+module Actions
+  module Pulp
+    class AbstractAsyncTaskGroup < Pulp::Abstract
+      include Actions::Base::Polling
+
+      # A call report (documented https://github.com/pulp/pulp/blob/master/docs/dev-guide/integration/rest-api/consumer/applicability.rst#id65)
+      # Looks like:  {
+      #     "_href": "/pulp/api/v2/task_groups/7744e2df-39b9-46f0-bb10-feffa2f7014b/",
+      #     "group_id": "7744e2df-39b9-46f0-bb10-feffa2f7014b"
+      #     }
+      #
+      #
+
```

Comment:
Could probably drop this blank line and the one `#` line above it to combine the two.